### DISCUSSION TITLE
Links as dropdown when reach the max container width available.

### DISCUSSION THREAD

**Discussion by nicolastakashi** [grafana/grafana#51957] [Dashboards, Panels & Visualizations]
**Steps to reproduce**:

Include a lot of links to the dashboard

**Actual Result**:

Nowadays, when we have a dashboard with a lot of links, it breaks down into a new line as the image below.

![image](https://user-images.githubusercontent.com/8205546/177487938-36d1d0ee-6307-4eef-a464-79c8b8e16872.png)

**Expected Result**

IMHO would be a better UI and UX if instead of breaking a new line, the links container be converted into a drop-down with the links inside.

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version: 8.5
- Data source type & version: N/A
- User OS & Browser: Linux/Google Chrome
- Others:

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
